src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -997,6 +998,9 @@
     <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="$(MicrosoftIdentityModelProtocolsOpenIdConnectVersion)" />
     <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" Version="$(MicrosoftIdentityModelJsonWebTokensVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="microsoft.data.sqlclient\netcore\src\microsoft\data\sqlclientx\handlers\connection\" />
```

Comment:
Something seems off about the casing of these folder names. Can you mimic the actual casing on disk? Don't want to deal with any compiler issues on Unix (just in case there are any problems)

Reply:
What IDE did you use to add this file/folder? Can you remove and add again via Visual Studio 2022?